app/assets/stylesheets/active_admin/components/_tables.scss

```diff
@@ -1,110 +1,106 @@
-//= depend_on_asset "active_admin/orderable.png"
-// ----------------------------------- Tables
+table {
+  margin-bottom: 1.4em;
+  width: 100%;
+}
 
-table tr {
-  td {
-    vertical-align: top;
-  }
+th {
+  text-align: left;
 }
 
-// --------- Index Tables
+th,
+td {
+  border-bottom: 1px solid $table-border-color;
+  padding: $table-cell-padding;
+  vertical-align: top;
+}
 
-table.index_table {
-  width: 100%;
-  margin-bottom: 10px;
-  border: 0;
-  border-spacing: 0;
+thead {
+  th {
+    border-bottom: 2px solid $table-border-color;
+    vertical-align: bottom;
+  }
+}
 
+// -------------------------------------- Index Tables
+.index-table {
   th {
-    @include section-header;
-    border-right: none;
-    text-align: left;
-    padding-left: $cell-horizontal-padding;
-    padding-right: $cell-horizontal-padding;
-
-    a, a:link, a:visited {
-      color: $section-header-text-color;
-      text-decoration: none;
+    a,
+    a:link,
+    a:visited {
+      color: $text-color;
       display: block;
+      position: relative;
+      text-decoration: none;
       white-space: nowrap;
     }
 
     &.sortable a {
-      background: image-url('active_admin/orderable.png') no-repeat 0 4px; padding-left: 13px;
+      &:after,
+      &:before {
+        border-left: 4px solid transparent;
+        border-right: 4px solid transparent;
+        bottom: 100%;
+        content: "";
+        height: 0;
+        position: absolute;
+        right: 0;
+        top: calc(50% - 2px);
+        width: 0;
+      }
     }
 
-    &.sorted-asc a { background-position: 0 -27px; }
-    &.sorted-desc a {  background-position: 0 -56px;}
+    &.sorted-asc a {
+      padding-right: 11px;
 
-    &.sorted-asc, &.sorted-desc {
-      @include gradient(darken($secondary-gradient-start, 5%), darken($secondary-gradient-stop, 5%));
+      &:after { // upward pointing arrow
+        border-bottom: 4px solid;
+      }
     }
 
-    &:last-child {
-      border-right: solid 1px #d4d4d4;
-    }
+    &.sorted-desc a {
+      padding-right: 11px;
 
+      &:before { // downward pointing arrow
+        border-top: 4px solid;
+      }
+    }
   }
 
-  tr.even td { background: $table-stripe-color; }
-
-  tr.selected td {
-    background: $table-selected-color;
+  tbody tr:nth-of-type(odd) {
+    background-color: $table-stripe-color;
   }
 
-  td {
-    padding: 10px $cell-horizontal-padding 8px $cell-horizontal-padding;
-    border-bottom: 1px solid #e8e8e8;
-    vertical-align: top;
+  tbody tr.selected {
```

Comment:
Avoid qualifying class selectors with an element.<br>Selector should have depth of applicability no greater than 2, but was 3